_targets.R

```diff
@@ -2,6 +2,9 @@
 library(targets)
 library(tarchetypes)
 library(tibble)
+library(lubridate)
```

Comment:
Nothing needed to change here but can be aware of the slight difference between a library call here vs adding the package in `tar_option_set` . Here is is always loaded each time pipelines is run. In `tar_option_set` they only get loaded if a target is outdated and needs to get rebuilt. At least I think that's the difference.